package.json

```diff
@@ -25,6 +25,7 @@
     "forwarder": "node ./development/static-server.js ./node_modules/@metamask/forwarder/dist/ --port 9010",
     "dapp-forwarder": "concurrently -k -n forwarder,dapp -p '[{time}][{name}]' 'yarn forwarder' 'yarn dapp'",
     "test:unit": "mocha --exit --require test/env.js --require test/setup.js --recursive './app/**/*.test.js'",
+    "test:unit:debug": "yarn test:unit --inspect-brk",
```

Comment:
@muinmomin Can you remove this from this PR?

You can add it through another PR if you'd like

Reply:
removed!